### ISSUE TITLE
Feature Request: Choose which calendars should be synchronized

### ISSUE THREAD

**Issue by larsomator** [balbusm/xul-ext-eds-calendar#17]
I have several calendars integrated in my lightning, some are just for information, and not with my dates.
It would be nice to be able to choose which calendars should be shown, so I can view my own events in the gnome-clock, and all information in Thunderbird/Lightning.

**Comment by balbusm** [comment]
It's a good idea. I will think about implementing it.

Thanks.

**Comment by LouisPM** [comment]
I would second larsomator's request.
Personally I have a good dozen of calendars. Some of them include just some information for documentation purposes (e.g. from when to when I took certain medications, etc.). But I don't need this kind of information to crowd the evolution calender panel of the shell.

Regardless, I highly appreciate your efforts. And I was happy to see that the integration with gnome v. 3.36 seems to be working nicely.

**Comment by andrei-a-papou** [comment]
I agree this would be a useful feature. As a temporary workaround, one could untick a calendar in the GNOME Calendar app and it wouldn't show notifications in the topbar.

P.S. Excellent addon, much appreciated! I use a Microsoft Exchange calendar via DavMail in Thunderbird, and this gives me the ability to push those events to GNOME's topbar applet. Great stuff and keep up the good work!
